programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -284,6 +287,9 @@ pub fn register_syscalls<'a>(
     // Log data
     syscall_registry.register_syscall_by_name(b"sol_log_data", SyscallLogData::call)?;
 
+    // Big mod exp
+    syscall_registry.register_syscall_by_name(b"sol_big_mod_exp", SyscallBigModExp::call)?;
```

Comment:
Can you register the syscall behind a feature gate? You can use a temporary address, which I can update afterwards with a valid key. You can refer to [this](https://github.com/solana-labs/solana/pull/27961/files#diff-ed9d8e9c9bc044a65944d919be16361b1969bca5122e281ff644a0bb4cbfa198R546) as reference.

Reply:
fixed